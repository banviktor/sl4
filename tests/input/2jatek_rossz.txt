start_game 2
start_game 4
exit