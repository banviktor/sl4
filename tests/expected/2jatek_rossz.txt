Pálya betöltve.	
Új játék elkezdve 2 játékossal.
Ez a(z) 1. kör.
A Piros Robot következik.
Folyamatban van egy másik játék, nem lehet újat kezdeni.